vuepress/docs/next/tutorials/getting-started/eks-install.md

```diff
@@ -89,32 +89,47 @@ See [Creating an Amazon EKS Cluster](https://docs.aws.amazon.com/eks/latest/user
 6. For `Node Group network configuration`, the subnets should already be set up and selected
 7. Select `Configure SSH access to nodes`. Follow the links to create a new SSH key pair if you don't already have one.
 8. Select `All` to allow all source IPs
-9. Click `Next`   
+9. Click `Next`
 10. Review your settings and then click `Create`. It may take a few minutes for the Node Group to be created.
 
+### Add the EBS CSI Add-on
+Starting with K8s 1.23, EKS requires an add-on in order to enable persistent volumes. The following instructions are for EBS CSI. See [the EBS CSI guide](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) and especially [the instructions for creating the EBS CSI IAM Role](https://docs.aws.amazon.com/eks/latest/userguide/csi-iam-role.html) for additional information.
+
+1. Determine the OpenID Connect provider URL for the cluster by going to `Clusters`→ YOUR-CLUSTER-NAME → `Overview`. It will be similar to this: 
+```
+https://oidc.eks.us-east-1.amazonaws.com/id/1C39B525EC0971750179719649SAMPLE
+```
+
+2. Follow [the AWS instructions](https://docs.aws.amazon.com/eks/latest/userguide/csi-iam-role.html) to prepare the IAM role required for the EBS CSI Add-on to function correctly. You will need to setup a new Role, e.g. YOUR-EBS-ROLE, with an OpenID Connect Identity Provider for your cluster using the URL from the previous step.
```

Comment:
add-on (lowercase) unless EBS CSI Add-on is a proper noun

set up

role should be lowercase unless preceded by the proprietary adj ??

i can read this two ways... reword slightly for clarity?? "You will need to set up a new role for your cluster, e.g. YOUR-EBS-ROLE, with an OpenID Connect Identity Provider and using the URL from the previous step." or "You will need to setup a new Role, e.g. YOUR-EBS-ROLE, with your cluster's OpenID Connect Identity Provider using the URL from the previous step."

Reply:
Tweaked a bit. I think EBS CSI Add-on is proper in this case. In the UI Add-on is generally capitalized when it refers to a specific add-on.